music_assistant/server/providers/deezer/__init__.py

```diff
@@ -83,6 +92,24 @@
 DEEZER_APP_SECRET = app_var(7)
 
 
+async def update_access_token(
+    app_id: str, app_secret: str, code: str, http_session: ClientSession
+) -> str:
+    """Update the access_token."""
+    response = await http_session.post(
+        "https://connect.deezer.com/oauth/access_token.php",
+        json={"code": code, "app_id": app_id, "secret": app_secret},
```

Comment:
```suggestion
        params={"code": code, "app_id": app_id, "secret": app_secret},
```
Otherwise it does not work for me on a clean installation. Until now we set `params` and `json`, but params seems to be good enough.

Reply:
Oh yeah forgot to test it, I changed it and then wanted to test if it worked later but forgot..

Reply:
Random question: Is it the reviewer or the pr author that is supposed to click "Resolve conversation"? :laughing: Till now i just did it myself but i guess the reviewer should be the one to decide whether it has been resolved... I dont know, im still fairly new to all of this i guess

Reply:
In most cases I do it the same way. I think it highly depends on whether you apply a requested change. If you do so, I would say it's okay to resolve it on your own. If not, you should answer the comment and discuss to find the best solution before someone resolves the comment

Reply:
Alright thanks 🙏